src/components/HomePopularProductCard.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import '../styles/index.scss';
```

Comment:
why are you importing this yet it is imported in the index.js. Once it is imported there, it affects every component, therefore, you do not need to import it in the components